search_params.py

```diff
@@ -171,13 +174,11 @@ def prepare_retrain_config(best_config, best_log_dir, retrain):
         best_config.merge_train_val = False
 
 
-def load_static_data(config, merge_train_val=False):
```

Comment:
Are these code changes necessary in this PR?

Reply:
Not really...But the argument has already be included in config